logger/logger.py

```diff
@@ -31,7 +31,7 @@ def __init__(self) -> None:
         self.logger = logging.getLogger("plugin_runner_logger")
         self.logger.setLevel(logging.INFO)
 
-        log_prefix = os.getenv("HOSTNAME", "")
+        log_prefix = f"'source_host': {os.getenv('HOSTNAME', '')}, 'container_index': {os.getenv('APTIBLE_PROCESS_INDEX', '')}"
```

Comment:
just for the sake of saving space I would do this like:

```suggestion
        log_prefix = f"{os.getenv('HOSTNAME', '?')}: {os.getenv('APTIBLE_PROCESS_INDEX', '?')}"
```

so it's stored as `host:n`... very verbose the other way and that's a lot of disk space